tests/test_llm_mock.py

```
@@ -0,0 +1,65 @@
+from datetime import datetime, timezone
+from typing import cast
+from unittest.mock import MagicMock
+
+import pytest
+
+from src.models import Message
+
+
+@pytest.mark.asyncio
+async def test_generic_honcho_llm_call_mock():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_generic_honcho_llm_call_mock`

Add return type annotation: `None`

(ANN201)